src/tester.py

```diff
@@ -40,10 +42,10 @@ def eval_list(glabels, slabels):
 def train_and_eval(classifier, trainfile, devfile, testfile, run_id):
     print(f"\nRUN: {run_id}")
     print("  %s.1. Training the classifier..." % str(run_id))
-    classifier.train(trainfile)

```

Comment:
The professor said it was very important to keep the tester.py file unmodified. 
Maybe you just changed it during the test, but before merging it into master, I think it is relevant to remove arguments that did not belong to the original file and to change them directly the classifier.py.